site/en/docs/web-platform/launch-handler/index.md

```diff
@@ -110,11 +114,12 @@ You can see a demo of the Launch Handler API in action in the
 Launch Handler API.
 
 1. Install the _Musicr 2.0_ app on a ChromeOS device.
+1. Open the app.
 1. Send yourself a link in a chat application of the form
    `https://launch-handler.glitch.me?track=https://example.com/music.mp3`. (You can customize
    `https://example.com/music.mp3` for any URL pointing to an audio file, for example,
    `https://launch-handler.glitch.me?track=https://cdn.glitch.me/3e952c9c-4d6d-4de4-9873-23cf976b422e%2Ffile_example_MP3_700KB.mp3?v=1638795977190`).
-1. Click the link in your chat app and notice how _Musicr 2.0_ opens and plays the track.
+1. Click the link in your chat app and notice how the link is handled by the existing instance of _Musicr 2.0_ and plays the track.
```

Comment:
I am not seeing this behavior on my ChromeOS device (114).
With the PWA installed and the app open, I sent myself the example link in Slack, clicked it and it opened a new tab in the browser, ignoring the open App.